sklearn/preprocessing/tests/test_data.py

```diff
@@ -1651,6 +1651,18 @@ def test_robust_scaler_zero_variance_features():
     assert_array_almost_equal(X_trans_new, X_expected_new, decimal=3)
 
 
+def test_robust_scaler_unit_variance():
+    # Check RobustScaler with unit_variance=True is equivalent to
+    # StandardScaler on large normal data
+    rng = np.random.RandomState(0)
+    X = rng.randn(100000, 1) * 5 + 2
+    robust_scaler = RobustScaler(unit_variance=True)
+    standard_scaler = StandardScaler()
+    assert_array_almost_equal(robust_scaler.fit_transform(X),
```

Comment:
We usually would use `assert_allclose` now instead of `assert_array_almost_equal`

Reply:
Changed though I remember how different the results are now. The results are only close with a large random dataset.

Reply:
ping @glemaitre